impl/src/main/java/org/pentaho/di/trans/dataservice/optimization/cache/CachedService.java

```diff
@@ -23,6 +23,8 @@
 package org.pentaho.di.trans.dataservice.optimization.cache;
 
 import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.MoreObjects;
```

Comment:
duplicate import `MoreObjects;` in CacheService.java

Reply:
fixed with https://github.com/pentaho/pdi-dataservice-server-plugin/commit/2c34eb09172284f7c60294657f4601b84e8fdc0c